v2/cn/linux-packages.md

```diff
@@ -270,14 +270,23 @@ sudo apt-get -y install --no-install-recommends openresty
 你可以在你的 CentOS 系统中添加 `openresty` 仓库，这样就可以便于未来安装或更新我们的软件包（通过 `yum check-update` 命令）。
 运行下面的命令就可以添加我们的仓库（对于 CentOS 8 或以上版本，应将下面的 `yum` 都替换成 `dnf`）：
 
+- CentOS 8 或者更老版本
```

Comment:
Why markdown list? Should use `## xxx` sections instead. Please fix other similar places.